.eleventy.js

```diff
@@ -1,18 +1,47 @@
 module.exports = (eleventyConfig) => {
+  eleventyConfig.addPassthroughCopy(".htaccess");
+  eleventyConfig.addTransform("wiki-links", function (content, outputPath) {
+    if (outputPath.endsWith(".html")) {
+      // We remove outer brackets from links
+      let output = content.replace(/(\[+(\<a(.*?)\<\/a\>)\]+)/g, "$2");
+      return output;
+    }
+    return content;
+  });
+
+  let markdownIt = require("markdown-it");
+  let markdownItReplaceLink = require("markdown-it-replace-link");
+  let markdownItOptions = {
+    html: true,
+    replaceLink: function (link, env) {
+      const isRelativePattern = /^(?!http|\/).*/;
+      const lastSegmentPattern = /[^\/]+(?=\/$|$)/i;
```

Comment:
It's interesting that these make the URLs on the homepage be "/" but on the "inbox" page the redirect does work for making the "/bubbles/inbox/todo" become "/bubbles/todo".

Reply:
I've totally missed that, I'll check on it.